atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/Event.java

```diff
@@ -41,5 +41,14 @@ static Event fromKeywordMap(Map<Keyword, ?> map) {
         return new ObjectMapper().convertValue(convertedMap, Event.class);
     }
 
+    static Map<Keyword, Object> toKeywordMap(Event event) {
+        Map<String, Object> stringToObject = new ObjectMapper().convertValue(event, Map.class);
+        Map<Keyword, Object> convertedMap = new HashMap<>();
+        EntryStream.of(stringToObject)
```

Comment:
Nice, four lines saved. @SerialVelocity, why would null values be a problem? Because of null being passed to Keyword.intern?